src/main/java/com/spikes2212/command/drivetrains/smartmotorcontrollerdrivetrain/CTRESmartMotorControllerDrivetrain.java

```diff
@@ -153,8 +153,10 @@ public void pidSet(UnifiedControlMode controlMode, double leftSetpoint, double r
                        FeedForwardSettings feedForwardSettings, TrapezoidProfileSettings trapezoidProfileSettings) {
         configPIDF(leftPIDSettings, rightPIDSettings, feedForwardSettings);
         configureTrapezoid(trapezoidProfileSettings);
-        leftMaster.set(controlMode.getCTREControlMode(), leftSetpoint);
-        rightMaster.set(controlMode.getCTREControlMode(), rightSetpoint);
+        leftMaster.set(controlMode.getCTREControlMode(), leftSetpoint, DemandType.ArbitraryFeedForward,
+                feedForwardSettings.getkS() * Math.signum(leftSetpoint));
+        rightMaster.set(controlMode.getCTREControlMode(), rightSetpoint, DemandType.ArbitraryFeedForward,
+                feedForwardSettings.getkS() * Math.signum(rightSetpoint));
```

Comment:
what if my position is `-600` and my setpoint is `-300`, would the `Math.signum` work then?